src/main/webapp/app/exercises/shared/exercise/exercise.component.ts

```diff
@@ -12,7 +12,9 @@ export abstract class ExerciseComponent implements OnInit, OnDestroy {
     private eventSubscriber: Subscription;
     @Input() embedded = false;
     @Input() course: Course;
+    @Input() isInExerciseGroup?: boolean;
```

Comment:
I would set this to false as a default and only set the field from the exercise group html. That way you do not have to check for null and can simplify the expression in the htmls: `*ngIf="isInExerciseGroup"`
```suggestion
    @Input() isInExerciseGroup = false;
```

Reply:
I made the change accordingly